packages/ui-buttons/src/BaseButton/index.tsx

```diff
@@ -131,6 +128,20 @@ class BaseButton extends Component<BaseButtonProps> {
     return this.interaction === 'enabled'
   }
 
+  // TODO: delete once the type of tabIndex is changed to number only
+  get tabIndex() {
```

Comment:
The HTML tabIndex accepts numbers only, so I decided to deprecate the string version.